fs-api/src/System/FS/API.hs

```diff
@@ -45,6 +44,12 @@ import           Util.CallStack
   Record that abstracts over the filesystem
 ------------------------------------------------------------------------------}
 
+-- | Abstract interface for performing file I\/O
+--
+-- [User-supplied buffers]: For functions that require user-supplied buffers
```

Comment:
I think I would make this into named fragment with an anchor that you can link to. Or at least an anchor

Reply:
TIL that anchoring/linking works pretty well with these types of definition lists! Thanks for the suggestion